libs/application/templates/parental-leave/src/fields/Review/Review.tsx

```diff
@@ -320,30 +321,57 @@ export const Review: FC<ReviewScreenProps> = ({
               error={hasError('payments.bank')}
             />
 
-            <GridRow marginTop={3} marginBottom={3}>
-              <GridColumn span={['12/12', '12/12', '12/12', '6/12']}>
-                <SelectController
-                  label={formatMessage(
-                    parentalLeaveFormMessages.shared.salaryLabelPensionFund,
-                  )}
-                  name="payments.pensionFund"
-                  id="payments.pensionFund"
-                  options={pensionFundOptions}
-                  defaultValue={pensionFund}
-                  onSelect={(s) =>
-                    setStateful((prev) => ({
-                      ...prev,
-                      pensionFund: s.value as string,
-                    }))
-                  }
-                  error={hasError('payments.pensionFund')}
-                />
-              </GridColumn>
+            <SelectController
+              label={formatMessage(
+                parentalLeaveFormMessages.shared.salaryLabelPensionFund,
+              )}
+              name="payments.pensionFund"
+              id="payments.pensionFund"
+              options={pensionFundOptions}
+              defaultValue={pensionFund}
+              onSelect={(s) =>
+                setStateful((prev) => ({
+                  ...prev,
+                  pensionFund: s.value as string,
+                }))
+              }
+              error={hasError('payments.pensionFund')}
+            />
+
+            <Label>
+              {formatMessage(parentalLeaveFormMessages.shared.unionName)}
+            </Label>
+
+            <Stack space={1}>
+              <RadioController
+                id="useUnion"
+                name="useUnion"
+                defaultValue={useUnion}
+                split="1/2"
+                options={[
+                  {
+                    label: formatMessage(
+                      parentalLeaveFormMessages.shared.yesOptionLabel,
+                    ),
+                    value: YES,
+                  },
+                  {
+                    label: formatMessage(
+                      parentalLeaveFormMessages.shared.noOptionLabel,
+                    ),
+                    value: NO,
+                  },
+                ]}
+                onSelect={(s: string) => {
+                  setStateful((prev) => ({
+                    ...prev,
+                    useUnion: s as YesOrNo,
```

Comment:
This probably applies for the other conditional rendering as well (the one blow)... but when no is selected, you've to manually reset the states of relevant inputs. ... so basically, you need to set this as null.

How to replicate?
1. radio = yes
2. select something in select box
3. radio = no
4. continue
5. now, union is set even though the user specified no.

Reply:
true, done